### PR TITLE
Add document for session variables.

### PR DESCRIPTION
**Pull request by morningman**
We need to document to explain those session variables Doris supported.
And also, when set a variables globally, we should also make it effective in current session.

ISSUE: #2283

### CODE REVIEW

File: docs/documentation/cn/administrator-guide/session-variables.md [hunk 1]
```diff
@@ -0,0 +1,280 @@
+# 变量
```
Comment (imay) [inline docs/documentation/cn/administrator-guide/session-variables.md]: license header

**Review by imay — CHANGES_REQUESTED** [review]

File: docs/documentation/cn/administrator-guide/session-variables.md (hunk 1 above)
Comment (morningman) [reply docs/documentation/cn/administrator-guide/session-variables.md]: Added

**Review by imay — DISMISSED** [review]
LGTM

**Review by imay — APPROVED** [review]
LGTM

**Pull request merged**